README.md

```diff
@@ -57,6 +57,11 @@ This README documents the current strawman architecture.  Changes must be made v
    1. Making categories in a query more specific. e.g. responding to a query for a biolink:NamedThing with a particular biolink:ChemicalSubstance.
    1. Making predicates more specific, e.g. responding to a query for “affects expression of” with an edge with predicate “increases expression of”.  In the response, the more specific edge must occur in the knowledge_graph portion of the response, and in individual results, that more specific edge will be bound to the less specific query edge.  Query Graph and Knowledge Graph edges need not match in either predicate or direction to be bound in an answer.
    1. Inverting symmetric predicates, e.g. if the KP contains information that A and B are correlated, then it should respond with that information whether the query is asked in the form A-[correlated_with]->B or B-[correlated_with]->A.
+1. Query Modes:
+   1. As described in the TRAPI specification, edges may be queried in either "lookup" or "inferred" mode.
+   1. KPs and ARAs must respond to lookup queries by treating the query as an exact database match
+   1. ARAs must respond to inferred mode one-hops with relevant results beyond an exact database match; KPs may also provide this capability
```

Comment:
Perhaps pedantic, but what should be the behavior of an ARA that does not support inferred mode for a particular query? It is required by this to do something it cannot do. Should it error out and do nothing? Or return exact database matches and no inferred results? Which seems like a MAY for ARAs then too?

Reply:
Agreed that 'may' sounds much more appropriate.
Alternatively, ARAs could invoke ChatGPT and ask it to make something up so they can respond. ;)